tests/rptest/tests/schema_registry_test.py

```diff
@@ -883,6 +892,53 @@ def test_config(self):
         result_raw = self._get_config_subject(subject=f"{topic}-key")
         assert result_raw.json()["compatibilityLevel"] == "BACKWARD_TRANSITIVE"
 
+        global_config = self._get_config().json()
+
+        result_raw = self._delete_config_subject(subject=f"{topic}-key")
+        assert result_raw.json(
+        )["compatibilityLevel"] == "BACKWARD_TRANSITIVE", f"{json.dumps(result_raw.json(), indent=1)}"
```

Comment:
nit: Maybe change the compatibility of the schema and then verifying it goes to the cluster config after the `delete config/` call

Reply:
Being pedantic but ensures that that's whats happening

Reply:
surprisingly, `delete /config/{subject}`  is expected to return the old compatibility level, so the same thing the last `get /config/{subject}` returned (above). All the more reason to make it explicit though, will do.

Reply:
oh my bad, i must have misinterpreted the documentation I was reading....